src/home_robot_hw/home_robot_hw/env/stretch_pick_and_place_env.py

```diff
@@ -19,6 +19,7 @@
     STRETCH_ARM_EXTENSION,
     STRETCH_ARM_LIFT,
     STRETCH_HOME_Q,
+    STRETCH_POSTNAV_Q,
```

Comment:
What is difference between postnav and pregrasp?

Reply:
POSTNAV_Q is more similar to navigation mode. The place and gaze policies were trained with a tilt of -0.78, while nav ends with a tilt of -0.52. 

In PREGRASP_Q, the camera faces the arm while in POSTNAV_Q it faces the base.